lib/features/steps/request_assertion_steps.rb

```diff
@@ -55,6 +55,16 @@ def assert_received_requests(request_count, list, list_name)
   list.sort_by_sent_at! request_count
 end
 
+# Verify that at least a certain amount of requests have been received
+# This step is only intended for use in stress tests
+#
+# @step_input min_received [Integer] The minimum amount of requests required to pass
+# @step_input request_type [String] The type of request (error, session, build, etc)
+Then('I have received at least {int} {word} requests') do |min_received, request_type|
+  list = Maze::Server.list_for(request_type)
+  assert(list.size >= min_received, "Actually received #{list.size} #{request_type} requests")
```

Comment:
`assert_operator` should give a slightly nicer diff if it fails:

```suggestion
  assert_operator(list.size, :>=, min_received, "Actually received #{list.size} #{request_type} requests")
```